src/telemetry/telemetry.c

```diff
@@ -400,6 +401,18 @@ build_telemetry_report()
 	ts_jsonb_add_str(parse_state,
 					 REQ_INSTALL_TIME,
 					 format_iso8601(ts_telemetry_metadata_get_install_timestamp()));
+	ts_jsonb_add_str(parse_state, REQ_INSTALL_METHOD, TIMESCALEDB_INSTALL_METHOD);
+
+	if (ts_version_get_os_info(&osinfo))
+	{
+		ts_jsonb_add_str(parse_state, REQ_OS, osinfo.sysname);
+		ts_jsonb_add_str(parse_state, REQ_OS_VERSION, osinfo.version);
+		ts_jsonb_add_str(parse_state, REQ_OS_RELEASE, osinfo.release);
+		if (osinfo.has_pretty_version)
+			ts_jsonb_add_str(parse_state, REQ_OS_VERSION_PRETTY, osinfo.pretty_version);
+	}
+	else
+		ts_jsonb_add_str(parse_state, REQ_OS, "Unknown");
```

Comment:
Since JSON is an open format I would suggest that if the field is missing, there is no information. Adding an "unknown" tag makes the format more complicated to process.

Reply:
This was there previously, but was removed by mistake. Keeping for compatibility.

Reply:
I do not feel strongly about this, feel free to ignore my comment. It's just an annoyance when you have to type something like the below to avoid reading something that is not defined (not including the need to handle typos or case).

```python
if 'os' not in json and json['os'] != 'Unknown':
   ...
```

I also have noticed that there is a tendency to add a lot of different strings to denote "unknown" so it snowballs into complicated checks of a assorted set of strings with different case and spelling because of this.

Since you are changing the format anyway, do you need to worry about compatibility? Is anybody relying on that if `json['os']` has to be `Unknown` or it will break?